src/main/java/be/acara/events/controller/dto/LikeEventDto.java

```diff
@@ -0,0 +1,14 @@
+package be.acara.events.controller.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
```

Comment:
Do we really need all these lombok annotations? A getter and the allargsconstructor should be more than enough.